Crash when running on cluster with BoundServiceAccountTokenVolume feature gate enabled

The BoundServiceAccountTokenVolume feature gate modifies service account tokens to be provided via a projected volume.  Permissions are slightly different for the token file itself:

```
-rw-------    1 root     root          1058 Dec 31 14:31 /var/run/secrets/kubernetes.io/serviceaccount/..data/token
```

Normally, the token file is mode 644.  The difference results in a crash:

```
2019/12/31 14:37:26 Starting sealed-secrets controller version: v0.9.6+dirty
panic: open /var/run/secrets/kubernetes.io/serviceaccount/token: permission denied

goroutine 1 [running]:
main.main()
	/home/travis/gopath/src/github.com/bitnami-labs/sealed-secrets/cmd/controller/main.go:271 +0x215
```

The fix is to set `securityContext.fsGroup` for the podspec to something like the following:

```yaml
spec:
  securityContext:
    fsGroup: 65534
```

This results in the token file having mode 640 with a group owner of 65534.  Any `fsGroup` value is acceptable, but this one is typically the `nobody` group.

--- a/controller-norbac.jsonnet
+++ b/controller-norbac.jsonnet
@@ -21,6 +21,9 @@ local namespace = 'kube-system';
     spec+: {
       template+: {
         spec+: {
+          securityContext+: {
+            fsGroup: 65534,
+          },
           containers_+: {
             controller: kube.Container('sealed-secrets-controller') {
               image: $.controllerImage,

